ietfdata/mailarchive.py

```diff
@@ -48,6 +49,68 @@ def _parse_archive_url(archive_url:str) -> Tuple[str, str]:
 
     return (mailing_list, message_hash)
 
+# =================================================================================================
+
+class IETFMessage:
```

Comment:
It seems like this is metadata about a message, rather than an extension of the message. Would it be more accurate to call it `MessageMetadata`?

Reply:
Yes; I think this is a better name. To maintain the separation between different datasources, would it make sense to move `MessageMetadata` into the `mailarchive_ext` module (proposed in another comment)?

Reply:
Yes, I think so.